### ISSUE TITLE
Stepper and Stepper Item - Provide alternative text for images - (2036570230)

### ISSUE THREAD

**Issue by dqateam** [Esri/calcite-components#5603]
### Module: 
 31 Stepper and Stepper Item 

### Violation: 
 Provide alternative text for images 

![image](https://user-images.githubusercontent.com/5023024/202738109-8bfdd038-8b99-46d0-8523-60c22e9d6180.png)

### WCAG Reference: 
- [1.1.1 Non-text Content](https://www.w3.org/WAI/WCAG21/Understanding/non-text-content.html) 

### Instance ID: 
 2036570230 

### Severity: 
 10 

### Description: 
 [Issue]
There are meaningful images without a textual equivalent. Examples include:
- Complete icon
- Error icon

[User Impact]
Screen reader users will be unable to determine what these images represent.

[Code Reference] 
 ```
<calcite-icon class="stepper-item-icon" aria-hidden="true" icon="checkCircleF" scale="s" calcite-hydrated=""></calcite-icon>
(...)
<svg class="svg" fill="currentColor" height="100%" viewBox="0 0 16 16" width="100%" xmlns="http://www.w3.org/2000/svg">
(...)
</svg>
 ``` 

### Note: 
 [Suggestion]
Ensure headings and labels are descriptive and unique. This includes the accessible names of buttons. Buttons that perform different actions must have different accessible names. The accessible name of a button can be set with internal text, an aria-label attribute, or an aria-labelledby attribute.

[Compliant Code Example] 
 ```
<calcite-icon class="stepper-item-icon" icon="checkCircleF" scale="s" calcite-hydrated=""></calcite-icon>
(...)
<svg class="svg" fill="currentColor" height="100%" viewBox="0 0 16 16" width="100%" xmlns="http://www.w3.org/2000/svg" role="img" aria-labelledby="title">
<title id="title">Complete</title>
(...)
</svg>
 ```
 

### Media Type: 
 Images 

### Additional Resources: 
- Report Source: Product accessibility evaluation conducted on the Calcite Design System in August, 2022.

**Comment by geospatialem** [comment]
Additional context is needed for assistive technology for when the `stepper-item` is `complete` or in `error`.

**Comment by geospatialem** [comment]
Mitigated in a PR prior to the 1.0.5 release, where the svg uses `aria-hidden="true"`.
